server/routes/release/actions.tsx

```diff
@@ -197,14 +197,14 @@ export default defineRoute(async (req, ctx) => {
 				<LinkWithMusicBrainz
 					entities={allArtists}
 					entityType='artist'
-					sourceEntityUrl={releaseUrl}
+					sourceEntityUrl={releaseUrl!}
```

Comment:
This assertion is not safe since we have no condition around this component at all, we need a condition on `releaseUrl` here.

P.S. I just saw that I previously had such a condition, although the `allArtists.map(...)` wrapper around the component is technically sufficient. (When there is no release, we have an empty artist array.)
The same construct for `allRecordings` did not have a condition but an assertion, so this is probably the inconsistency that you meant...